requirements.txt

```diff
@@ -0,0 +1,2 @@
+blessed~=1.18
+numpy~=1.21.0
```

Comment:
Should this be `numpy~=1.21`? For consistency + that's how the template readme says to do things.